packages/engine-render/src/components/sheets/extensions/row-header-layout.ts

```diff
@@ -15,24 +15,90 @@
  */
 
 import type { IScale } from '@univerjs/core';
+import { numberToABC } from '@univerjs/core';
 
 import { DEFAULT_FONTFACE_PLANE, FIX_ONE_PIXEL_BLUR_OFFSET, MIDDLE_CELL_POS_MAGIC_NUMBER } from '../../../basics/const';
 import { getColor } from '../../../basics/tools';
 import type { UniverRenderingContext } from '../../../context';
 import { SheetRowHeaderExtensionRegistry } from '../../extension';
 import type { SpreadsheetSkeleton } from '../sheet-skeleton';
+import type { IARowCfg, IARowCfgObj, IColumnStyleCfg, IRowStyleCfg } from '../interfaces.ts';
 import { SheetExtension } from './sheet-extension';
 
 const UNIQUE_KEY = 'DefaultRowHeaderLayoutExtension';
 
+export interface IRowsHeaderCfgParam {
+    headerStyle?: Partial<IRowStyleCfg>;
+    rowsCfg?: IARowCfg[];
+}
+
+const DEFAULT_ROW_STYLE = {
+    fontSize: 13,
+    fontFamily: DEFAULT_FONTFACE_PLANE,
+    fontColor: '#000000',
+    backgroundColor: getColor([248, 249, 250]),
+    borderColor: getColor([217, 217, 217]),
+    textAlign: 'center',
+    textBaseline: 'middle',
+} as const;
+
 export class RowHeaderLayout extends SheetExtension {
     override uKey = UNIQUE_KEY;
-
     override Z_INDEX = 10;
+    rowsCfg: IARowCfg[] = [];
+    headerStyle: IRowStyleCfg = {
+        fontSize: DEFAULT_ROW_STYLE.fontSize,
+        fontFamily: DEFAULT_ROW_STYLE.fontFamily,
+        fontColor: DEFAULT_ROW_STYLE.fontColor,
+        backgroundColor: DEFAULT_ROW_STYLE.backgroundColor,
+        borderColor: DEFAULT_ROW_STYLE.borderColor,
+        textAlign: DEFAULT_ROW_STYLE.textAlign,
+        textBaseline: DEFAULT_ROW_STYLE.textBaseline,
+    };
+
+    constructor(cfg?: IRowsHeaderCfgParam) {
+        super();
+        if (cfg) {
+            this.configHeaderRow(cfg);
+        }
+    }
+
+    configHeaderRow(cfg: IRowsHeaderCfgParam) {
+        this.rowsCfg = cfg.rowsCfg || [];
+        this.headerStyle = { ...this.headerStyle, ...cfg.headerStyle };
+    }
+
+    getCfgOfCurrentRow(rowIndex: number) {
+        let mergeWithSpecCfg;
+        let curRowSpecCfg;
+        const rowsCfg = this.rowsCfg || [];
 
+        if (rowsCfg[rowIndex]) {
+            if (typeof rowsCfg[rowIndex] == 'string') {
+                rowsCfg[rowIndex] = { text: rowsCfg[rowIndex] } as IARowCfgObj;
+            }
+            curRowSpecCfg = rowsCfg[rowIndex] as IRowStyleCfg & { text: string };
+            mergeWithSpecCfg = { ...this.headerStyle, ...curRowSpecCfg };
+        } else {
+            mergeWithSpecCfg = { ...this.headerStyle, text: numberToABC(rowIndex) };
```

Comment:
It would be preferable to display line numbers by default rather than letter numbering.